bitnami/mastodon/4/debian-11/rootfs/opt/bitnami/scripts/mastodon-env.sh

```diff
@@ -247,7 +248,7 @@ MASTODON_AWS_ACCESS_KEY_ID="${MASTODON_AWS_ACCESS_KEY_ID:-"${AWS_ACCESS_KEY_ID:-
 export MASTODON_AWS_ACCESS_KEY_ID="${MASTODON_AWS_ACCESS_KEY_ID:-}"
 export AWS_ACCESS_KEY_ID="$MASTODON_AWS_ACCESS_KEY_ID"
 export MASTODON_CFG_AWS_ACCESS_KEY_ID="$MASTODON_AWS_ACCESS_KEY_ID"
-export MASTODON_S3_REGION="${MASTODON_S3_REGION:-us-east-1}"
+export MASTODON_S3_REGION="${MASTODON_S3_REGION:-"${S3_REGION:-us-east-1}"
```

Comment:
There is a typo with a missing `}"` in this line. Could you please fix it?

Reply:
Sorry about that. I thought I had that right.

```suggestion
export MASTODON_S3_REGION="${MASTODON_S3_REGION:-"${S3_REGION:-us-east-1}"}"
```